### ISSUE TITLE
Dark Style must be the default design and bugs fixed.

### ISSUE THREAD

**Issue by felisucoibi** [RetroShare/RetroShare#2053]
Dark Style must be the default design and bugs fixed.

It's better becuase make screns to use less energy so good for the planet, is also good because is less fatigue for the eyes.

So needs to be fixed to be perfect and usable and then set up default install with it.

The white one is from the past.

Also main icons shoud be big (6xx64) for actual screens by default.

**Comment by sehraf** [comment]
> It's better becuase make screns to use less energy so good for the planet, is also good because is less fatigue for the eyes.

That's not true. The power usage of the screen depends on the display technology and dark mode in only good in dark environments. Especially for long time reading, bright mode is preferable.

**Comment by chelovechishko** [comment]
> Especially for long time reading, bright mode is preferable.

Based on what you make such assumptions?

**Comment by sehraf** [comment]
> Especially for long time reading, bright mode is preferable.
> Based on what you make such assumptions?

Science? Google it. 
Of course this is also a matter of taste.

**Comment by chelovechishko** [comment]
> science

Ah-h-h, get it
https://www.nature.com/articles/s41598-018-28904-x
> we found that the choroid, the heavily perfused layer behind the retina in the eye, becomes about 16 µm thinner in only one hour when subjects read black text on white background but about 10 µm thicker when they read white text from black background. Studies both in animal models and in humans have shown that thinner choroids are associated with myopia development and thicker choroids with myopia inhibition. Therefore, reading white text from a black screen or tablet may be a way to inhibit myopia, while conventional black text on white background may stimulate myopia.

**Comment by sehraf** [comment]
https://www.wired.co.uk/article/dark-mode-chrome-android-ios-science

> Anna Cox, a professor of human-computer interaction at UCL, dispute this is the case, saying, “I’m not aware of any robust evidence that white text on black background reduces eye strain”.
> However, the effort of looking at a screen can vary a lot depending on your surroundings.

My point was not to start a discussion about what is wrong and what is right. My point is that bluntly demanding `Dark Style __must__ be the default` based on arguable evidence is not the way to go. 

I personally would solve it like others do: when starting rs for the first time simply ask the user to select a style.

**Comment by chelovechishko** [comment]
Full white on black screen is not good and per my organoleptic "studies".

> My point is that bluntly demanding Dark Style __must__ be the default based on arguable evidence is not the way to go.

And demanding of White Style be the default just because is the way, ho-ho-ho?

My point is: i am indifferent.

> when starting rs for the first time simply ask the user to select a style.

Were "external" stylesheets installed by default (in $HOME/.retroshare/~~stylesheets~~qss)? I don't remember (though, i totally do not need it, i have only qss that i, let say, want).
Although, i guess, if a person reached "appearance settings", it will manually decide what the look should be.

**Comment by csoler** [comment]
I agree with Sehraf about keeping white as the default. That being said, black should work properly since some users (apparently) like it too.

**Comment by felisucoibi** [comment]
As always do what you want, but i'm not usually wrong... is something that happens... meanwhile everything is moving to have a good dark style experience, facebook moved to to dark style, android too, whatsapp, twitter, ubuntu, and i liked the link of @chelovechishko 
"Therefore, reading white text from a black screen or tablet may be a way to inhibit myopia, while conventional black text on white background may stimulate myopia."
https://www.nature.com/articles/s41598-018-28904-x#Abs1

Epic games, steam.... all video editors and audio editors.... netflix, hulu, apple itunes,

But as always do what you want... rs with white skin looks like something from the past.

**Comment by felisucoibi** [comment]
Callibre just have its new brand new dark style working... rs has a crappy dark style :p

**Comment by csoler** [comment]
If you have some time, please fix it.

**Comment by felisucoibi** [comment]
> If you have some time, please fix it.

trust me, is something as anoying that i really tried.... is the default blue link and is not a qss style problem but a more global problem becasue all the styles use the defalut link blue, so is something deep and hide inside rs code that is by far away from my skills qith c++ and qss.

**Comment by hardBSDk** [comment]
@csoler Where I can found the dark mode?

I'm using GNOME 41 with AppImage build, the dark mode with GTK+ doesn't appear (I use dark mode on GNOME).

**Comment by defnax** [comment]
@csoler close
